dexter/src/main/java/com/karumi/dexter/CollectionUtils.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (C) 2015 Karumi.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.karumi.dexter;
+
+import java.util.Collection;
+
+/**
+ * Utility class containing useful methods related to the {@link Collection} classes
+ */
+final class CollectionUtils {
+
+  /**
+   * Returns the first element of the collection or null if there is not any
+   */
+  public static <T> T getFirstFromCollection(Collection<T> items) {
```

Comment:
Instead of create these kind of methods why don't you use just a list on the methods needed like ``onPermissionRationaleShouldBeShown``?